ssl/statem/extensions_srvr.c

```diff
@@ -753,6 +766,32 @@ int tls_parse_ctos_psk(SSL *s, PACKET *pkt, unsigned int context, X509 *x,
     }
 
     sess->ext.tick_identity = id;
+
+    now = (uint32_t)time(NULL);
+    agesec = now - (uint32_t)sess->time;
```

Comment:
I don't think it makes a difference - even if it wraps I think agesec still works out right? Or am I confused.